test/functions/blockchainFunctionsTest.js

```diff
@@ -155,99 +154,74 @@ describe('tokenTopTransactions_', () => {
   })
 })
 
-describe('ethTopTransactions_', () => {
```

Comment:
You need rebase master.

Reply:
Yes, I saw, that it was a bit strange, it is fixed now.